update_chroot

```diff
@@ -97,15 +98,15 @@ EOF
 
 sudo_clobber "/etc/portage/repos.conf/coreos.conf" <<EOF
 [DEFAULT]
-main-repo = portage-stable
+main-repo = gentoo
 
-[gentoo]
-disabled = true
+#[gentoo]
+#disabled = true
 
 [coreos]
 location = ${COREOS_OVERLAY}
 
-[portage-stable]
+[gentoo]
```

Comment:
We don't use the upstream gentoo repo. Everything is vendored into portage-stable so that we can preserve the ebuilds independently of upstream.